airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle

```diff
@@ -48,152 +27,28 @@ jsonSchema2Pojo {
-    testFixturesImplementation 'org.junit.jupiter:junit-jupiter-params:5.4.2'
-
-    // From source-jdbc
-    implementation 'org.apache.commons:commons-lang3:3.11'
-    testImplementation libs.postgresql
-    integrationTestJavaImplementation libs.testcontainers.postgresql
-    testFixturesImplementation libs.airbyte.protocol
-    // todo (cgardens) - the java-test-fixtures plugin doesn't by default extend from test.
-    //  we cannot make it depend on the dependencies of source-jdbc:test, because source-jdbc:test
-    //  is going to depend on these fixtures. need to find a way to get fixtures to inherit the
-    //  common test classes without duplicating them. this should be part of whatever solution we
-    //  decide on for a "test-java-lib". the current implementation is leveraging the existing
-    //  plugin, but we can something different if we don't like this tool.
-    testFixturesRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.4.2'
-    testFixturesImplementation group: 'org.mockito', name: 'mockito-junit-jupiter', version: '4.0.0'
-
-    // From `standard-source-test`:
-    testFixturesImplementation 'org.mockito:mockito-core:4.6.1'
-    testFixturesRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.4.2'
-    testFixturesImplementation 'org.junit.jupiter:junit-jupiter-api:5.4.2'
-    testFixturesImplementation 'org.junit.jupiter:junit-jupiter-params:5.8.1'
-
-    // From `airbyte-test-utils`:
-    // api project(':airbyte-db:db-lib')
-    testFixturesImplementation 'io.fabric8:kubernetes-client:5.12.2'
-    testFixturesImplementation libs.temporal.sdk
-    testFixturesApi libs.junit.jupiter.api
-    // Mark as compile only to avoid leaking transitively to connectors
-    testFixturesCompileOnly libs.testcontainers.jdbc
-    testFixturesCompileOnly libs.testcontainers.postgresql
-    testFixturesCompileOnly libs.testcontainers.cockroachdb
-    testFixturesImplementation libs.testcontainers.cockroachdb
-}
-
-def getFullPath(String className) {
-    def matchingFiles = project.fileTree("src/testFixtures/java")
-            .filter { file -> file.getName().equals("${className}.java".toString()) }.asCollection()
-    if (matchingFiles.size() == 0) {
-        throw new IllegalArgumentException("Ambiguous class name ${className}: no file found.")
-    }
-    if (matchingFiles.size() > 1) {
-        throw new IllegalArgumentException("Ambiguous class name ${className}: more than one matching file was found. Files found: ${matchingFiles}")
-    }
-    def absoluteFilePath = matchingFiles[0].toString()
-    def pathInPackage = project.relativePath(absoluteFilePath.toString()).replaceAll("src/testFixtures/java/", "").replaceAll("\\.java", "")
-    return pathInPackage
-}
+    implementation project(':airbyte-cdk:java:airbyte-cdk:dependencies')
+    implementation project(':airbyte-cdk:java:airbyte-cdk:core')
 
-def generateSourceTestDocs = tasks.register('generateSourceTestDocs', Javadoc) {
```

Comment:
This task is the product of a lot of effort to do... something. Perhaps we could focus on making the code readable first before worrying about the documentation which is already stale anyway.